### PR TITLE
chore/dash-var-refresh: change default refresh to 2(time range)

### PR DESCRIPTION
**Pull request by mtulio**
Change the default variable refresh behavior from "On Dashboard Reload" to "On Time Range Change" on Grafana dashboards variables.

That is quite useful when navigating to old data, avoiding refresh the dashboards every new time range change.

https://github.com/openshift/cluster-monitoring-operator/pull/1097

### CODE REVIEW

File: contrib/mixin/mixin.libsonnet [hunk 1]
```diff
@@ -1287,7 +1287,7 @@
             name: 'cluster',
             options: [],
             query: 'label_values(etcd_server_has_leader, job)',
-            refresh: 1,
+            refresh: 2,
```
Comment (ptabor) [inline contrib/mixin/mixin.libsonnet:1290]: Please add a comment explaining what does the constant mean.

**Review by ptabor — CHANGES_REQUESTED** [review]

File: contrib/mixin/mixin.libsonnet (hunk 1 above)
Comment (mtulio) [reply contrib/mixin/mixin.libsonnet:1290]: @ptabor do you mean a inline comment on the code or here? On the PR description I've detailed the change:

Comment (ptabor) [reply contrib/mixin/mixin.libsonnet:1290]: In-line please. It requires a lot of knowledge to decode the constant when scanning the config.

Comment (mtulio) [reply contrib/mixin/mixin.libsonnet]: @ptabor great! I've create a const and set it as value 2, please let me know if it fits better. https://github.com/etcd-io/etcd/pull/12823/commits/048b90f7a7987eaa6bdd1912f46cd202c71936a6

**Comment by ptabor** [comment]
Thank you.

**Pull request merged**